components/notebook-controller/Makefile

```diff
@@ -25,6 +28,9 @@ check-license:
 
 # Run tests
 test: generate fmt vet manifests check-license
+	mkdir -p ${ENVTEST_ASSETS_DIR}
+	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/master/hack/setup-envtest.sh
+	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR);
```

Comment:
This won't work as it is, because make's default shell is `/bin/sh`.
See: https://stackoverflow.com/a/43566158